File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1181.py
n = int(input())
e = input()
soma=0.0
for i in range(12):
    for j in range(12):
        valor = float(input())
        if(i==n): soma+=valor

if(e=='S'): print(soma)
else: print("{:.1f}".format(soma/12))

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1181.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
    
n = parseInt(lines.shift())
e = lines.shift()
soma=0.0
for(i=0;i<12;i++){
    for(j=0;j<12;j++){
        valor = parseFloat(lines.shift())
        if(i==n){
            soma+=valor;
        }
    }
}

if(e=='S'){
    console.log(soma.toFixed(1))
}else{
    console.log((soma/12).toFixed(1))
} 
